### PR TITLE
[ID] Add stubs for input device articles

### PR DESCRIPTION
**Pull request by mousesalt**
**[/wiki/Gameplay/Input device](https://osu.ppy.sh/wiki/en/Gameplay/Input_device)**

### CODE REVIEW

File: wiki/Gameplay/Input_device/Beatmania_controller/id.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Kontroller Beatmania
+
+![](img/beatmania-controller.jpg "Kontroller Beatmania 7 tombol")
+
+**Kontroller Beatmania** adalah sebuah tipe dari kontroller gim yang cocok digunakan pada seri game *[Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")*.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Beatmania_controller/id.md:9]:
```suggestion
**Kontroller Beatmania** adalah sebuah tipe kontroller yang digunakan pada seri game *[Beatmania](https://id.wikipedia.org/wiki/Beatmania "Wikipedia")*.
```

File: wiki/Gameplay/Input_device/Beatmania_controller/id.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Karena sifatnya, kontroller Beatmania paling sering digunakan sebagai perangkat input di [osu!mania](/wiki/Game_mode/osu!mania).
+
+## Metode masukan
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Beatmania_controller/id.md:13]:
```suggestion
## Metode input
```

File: wiki/Gameplay/Input_device/Beatmania_controller/id.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Pemain-pemain yang memiliki kontroller Playstation 2 (PS2) *Beatmania*, dapat menggunakan adaptor untuk menghubungkan kontroller ke komputer mereka melalui USB dan mengatur tombol di opsi dalam permainan. Namun, bagi pemain yang memiliki pengontrol *DJ DAO IIDX*, cukup mencolokkan pengontrol ke komputer mereka melalui USB dan mengatur kunci dalam permainan.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Beatmania_controller/id.md:15]:
```suggestion
Pemain-pemain yang memiliki kontroller Playstation 2 (PS2) *Beatmania*, dapat menggunakan adaptor untuk menghubungkan kontroller ke komputer mereka melalui USB dan mengatur tombol di opsi pengaturan dalam permainan. Namun, bagi pemain yang memiliki kontroller *DJ DAO IIDX*, dapat tinggal mencolokkan kontroller ke komputer mereka melalui USB dan mengatur key dalam permainan.

File: wiki/Gameplay/Input_device/Dance_pad/id.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Dance pad
+
+![](img/dance-pad.jpg "kontroller dance pad 4 tombol")
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Dance_pad/id.md:7]:
```suggestion
![](img/dance-pad.jpg "Kontroller dance pad 4 tombol")
```

File: wiki/Gameplay/Input_device/Dance_pad/id.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+**Dance pad**, atau **Dance mat**, adalah sebuah tipe kontroller gim rhythm yang menggunakan penempatan kaki para pemain sebagai input. Ditunjukkan pada gambar diatas, dance pad ditempatkan pada lantai dan memungkinkan para pengguna untuk melangkah ke mana saja dari empat hingga sembilan "arah" untuk mengaktifkan tombol. Jenis kontroller ini paling sering digunakan pada gim-gim rhythm, seperti *[Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia")*.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Dance_pad/id.md:9]:
```suggestion
**Dance pad**, atau **Dance mat**, adalah sebuah tipe kontroller gim rhythm yang menggunakan penempatan kaki para pemain sebagai input. Ditunjukkan pada gambar di atas, dance pad ditempatkan pada lantai dan memungkinkan para pengguna untuk melangkah ke mana saja dari empat hingga sembilan "arah" untuk mengaktifkan tombol. Jenis kontroller ini paling sering digunakan pada gim-gim rhythm, seperti *[Dance Dance Revolution](https://id.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia")*.
```

File: wiki/Gameplay/Input_device/Dance_pad/id.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+[Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia") Dance Mat membutuhkan penggunaan adaptor kontroller [GameCube](https://en.wikipedia.org/wiki/GameCube "Wikipedia") untuk dapat berfungsi dengan komputer. Setelah komputer terhubung dengan dance pad, aturlah key yang benar di dalam pengaturan osu!.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Dance_pad/id.md:11]:
```suggestion
[Wii](https://id.wikipedia.org/wiki/Wii "Wikipedia") Dance Mat membutuhkan penggunaan adaptor kontroller [GameCube](https://id.wikipedia.org/wiki/GameCube "Wikipedia") untuk dapat digunakan pada komputer. Setelah dance pad terhubung dengan komputer, aturlah key yang benar di dalam pengaturan osu!.

File: wiki/Gameplay/Input_device/Drum_controller/id.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+# Kontroller drum
+
+![](img/taiko-drum.jpg "Drum Taiko")
+
+**Kontroller drum** atau *Drum controller* adalah sebuah perangkat input yang didesain untuk bekerja dengan gim-gim drum seperti Taiko. Input pada kontroller ini paling sering mempunyai empat area: pusat drum kanan, pusat drum kiri, rim drum kanan, dan pelek drum kiri, dengan setiap input (key) diaktifkan menggunakan sebuah stik drum stick yang dipukul-pukul pada area yang sesuai dari drum.
+
+## Input
+
+Pemilik kontroller drum Taiko untuk [PlayStation 4 (PS4)](https://en.wikipedia.org/wiki/PlayStation_4 "Wikipedia") atau [Nintendo Switch](https://en.wikipedia.org/wiki/Nintendo_Switch "Wikipedia") perlu untuk menancapkan kontroller ke komputer melalui USB dan mengatur key dalam pengaturan permainan osu!. Pemilik kontroller [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia") Tatacon perlu untuk pemasangan kontroller ke komputer sebelum menghidupkan opsi layanan Wiimote/Tatacon pada pengaturan osu!. Sebagian besar kontroller drum pihak ketiga juga akan bekerja dengan osu!, tetapi dukungan mungkin akan berbeda-beda.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Drum_controller/id.md:13]:
```suggestion
Pemilik kontroller drum Taiko untuk [PlayStation 4 (PS4)](https://id.wikipedia.org/wiki/PlayStation_4 "Wikipedia") atau [Nintendo Switch](https://id.wikipedia.org/wiki/Nintendo_Switch "Wikipedia") perlu untuk memasang kontroller ke komputer melalui USB dan mengatur key dalam pengaturan permainan osu!. Pemilik kontroller [Wii](https://id.wikipedia.org/wiki/Wii "Wikipedia") Tatacon perlu untuk memasangkan kontroller ke komputer sebelum menghidupkan opsi layanan Wiimote/Tatacon pada pengaturan osu!. Sebagian besar kontroller drum pihak ketiga juga akan bekerja dengan osu!, tetapi dukungan mungkin akan berbeda-beda.
```

File: wiki/Gameplay/Input_device/Graphics_tablet/id.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+# Graphics tablet
+
+*Kunjungi juga: [Gaya bermain](/wiki/Play_style)*
+
+![](img/tablet.jpg "Graphics tablet")
+
+**Graphics tablet**, atau *tablet*, adalah sebuah perangkat input yang awalnya ditujukan untuk gambar digital dan karya seni, tetapi dapat digunakan sebagai cara untuk mengontrol gerakan kursor pada mode permainan [osu!](/wiki/Game_mode/osu!). Menggunakan tablet merupakan salah satu gaya bermain yang paling umum pada mode permainan osu!.
+
+*Catatan untuk pemain tablet: penggunaan Tablet PC atau tablet klik di osu! terkadang tidak sesuai. Untuk memperbaikinya, aktifkan pengaturan `dukungan OSTablet PC` di menu Opsi.*
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Graphics_tablet/id.md:13]:
```suggestion
*Catatan untuk pemain tablet: penggunaan Tablet PC atau tablet klik di osu! terkadang tidak kompatibel. Untuk memperbaikinya, aktifkan pengaturan `dukungan OSTablet PC` di opsi menu Pengaturan.*
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
+---
+stub: true
+---
+
+# Mouse
+
+**Mouse** adalah cara yang sangat populer untuk bermain osu!. Kamu dapat menggunakan mouse untuk mengarahkan dan menekan. Tombol pada tengah mouse diatur untuk menghentikan permainan (dapat dinonaktifkan di dalam opsi), dan roda gulir dapat digunakan untuk mengubah volume.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:7]:
```suggestion
**Mouse** adalah cara yang sangat populer untuk bermain osu!. Kamu dapat menggunakan mouse untuk mengarahkan dan menekan. Tombol pada tengah mouse diatur untuk menghentikan permainan (dapat dinonaktifkan di dalam opsi pengaturan), dan roda gulir dapat digunakan untuk mengubah volume.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+Opsi ini mengizinkan untuk memilih kecepatan mouse yang umumnya diatur bernilai 1x.
+
+### Raw input
+
+Raw input akan melewati akselerasi mouse Windows dan memberikan gerakan mouse yang paling akurat. Raw input perlu diaktifkan untuk mengatur sensitivitas mouse pada Windows 7+.
+
+### Membatasi kursor mouse
+
+Opsi ini mengizinkan kursor mouse untuk berada tetap di dalam jendela osu! dengan mengatur pada `Only when fullscreen`. Opsi yang tersedia meliputi:
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:21]:
```suggestion
Opsi ini mengizinkan kursor mouse untuk berada tetap di dalam window osu! dengan mengatur pada `Only when fullscreen`. Opsi yang tersedia meliputi:
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+- `Never`: Mouse tidak akan berada tetap di dalam jendela osu!.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:23]:
```suggestion
- `Never`: Mouse tidak akan berada tetap di dalam window osu!.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+- `Only when fullscreen`: Mouse akan berada tetap di dalam jendela osu! ketika pada layar penuh.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:24]:
```suggestion
- `Only when fullscreen`: Mouse akan berada tetap di dalam window osu! ketika pada layar penuh.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+- `Always`: Mouse akan berada tetap di dalam jendela osu!.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:25]:
```suggestion
- `Always`: Mouse akan berada tetap di dalam window osu!.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+### Nonaktifkan roda mouse dalam mode bermain
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:27]:
```suggestion
### Nonaktifkan roda mouse selagi bermain
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+
+Selama bermain, Kamu dapat menggunakan roda mouse untuk menyesuaikan volume dan menjeda permainan. Mengaktifkan opsi ini akan menonaktifkan fungsionalitas ini saat bermain.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:29]:
```suggestion
Selama bermain, Kamu dapat menggunakan roda mouse untuk menyesuaikan volume dan menjeda permainan. Mengaktifkan opsi ini akan menonaktifkan fungsi ini saat bermain.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+### Nonaktifkan tombol mouse dalam mode bermain
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:31]:
```suggestion
### Nonaktifkan tombol mouse selagi bermain
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+
+Opsi ini akan menonaktifkan semua tombol mouse selama bermain, yang mungkin berguna bagi orang-orang yang menggunakan keyboard untuk menekan.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:33]:
```suggestion
Opsi ini akan menonaktifkan semua tombol mouse selama pemain sedang bermain, yang mungkin berguna bagi orang-orang yang menggunakan keyboard untuk menekan.
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+
+### Riak kursor
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:35]:
```suggestion
### Efek gelombang kursor
```

File: wiki/Gameplay/Input_device/Mouse/id.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+
+Kursor akan bergerak ke luar saat menekan.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Mouse/id.md:37]:
```suggestion
Kursor akan memunculkan sebuah gelombang tipis yang membesar saat ditekan.
```

File: wiki/Gameplay/Input_device/Touch_device/id.md [hunk 20]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Perangkat sentuh
+
+*See also: [Play style](/wiki/Play_style)*
+
+**Perangkat sentuh** atau *touch device* adalah sebuah perangkat layar sentuh yang mengizinkan pengguna untuk berinteraksi dengan komputer menggunakan jari-jari tangan. osu! mendukung perangkat sentuh asli. Kamu dapat mengentuh layar untuk mengarahkan dan menekan pada saat bersamaan. Perangkat sentuh paling sering muncul dalam bentuk laptop layar sentuh, tablet grafis kelas atas, atau smartphone.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Touch_device/id.md:9]:
```suggestion
**Perangkat sentuh** atau *touch device* adalah sebuah perangkat layar sentuh yang mengizinkan pengguna untuk berinteraksi dengan komputer menggunakan jari-jari tangan. osu! mendukung perangkat sentuh secara langsung. Kamu dapat menyentuh layar untuk mengarahkan kursor dan menekan pada saat bersamaan. Perangkat sentuh yang paling umum adalah laptop layar sentuh, tablet grafis kelas atas, atau smartphone.
```

File: wiki/Gameplay/Input_device/Wiimote/id.md [hunk 21]
```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Wiimote
+
+![](img/wiimote.jpg "Wiimote")
+
+**Wiimote** adalah sebuah kontroller utama untuk Nintendo [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia"). Meskipun bagi sebagian besar orang terlihat menarik perhatian atau konyol, Wiimote dapat digunakan di osu! untuk membidik menggunakan bilah sensor. Kamu dapat memasang kembali tombol apa pun untuk mengklik pada opsi osu!.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Wiimote/id.md:9]:
```suggestion
**Wiimote** adalah sebuah kontroller utama untuk Nintendo [Wii](https://id.wikipedia.org/wiki/Wii "Wikipedia"). Meskipun bagi sebagian besar orang kontroller ini terlihat menarik perhatian atau konyol untuk dimainkan di osu!, Wiimote dapat digunakan di osu! untuk membidik menggunakan bar sensor. Kamu dapat mengatur ulang tombol apa pun untuk mengklik pada opsi pengaturan osu!.
```

File: wiki/Gameplay/Input_device/Wiimote/id.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
...
+
+Wiimote mungkin juga dapat mengontrol pukulan drum pada [osu!taiko](/wiki/Game_mode/osu!taiko) melalui kontrol gerak atau tombol pada Wiimote itu sendiri. Namun, ini mungkin memerlukan pengetahuan lebih lanjut tentang perangkat lunak Wiimotes agar dapat berfungsi.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Wiimote/id.md:11]:
```suggestion
Wiimote bisa juga digunakan untuk mengontrol pukulan drum pada [osu!taiko](/wiki/Game_mode/osu!taiko) melalui kontrol gerak (motion) atau tombol pada Wiimote itu sendiri. Namun, ini mungkin memerlukan pengetahuan lebih lanjut tentang perangkat lunak Wiimote agar dapat berfungsi.
```

File: wiki/Gameplay/Input_device/Wiimote/id.md [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+
+![](img/wiimote-support-option.jpg "Opsi dukungan Wiimote pada osu!")
+
+Agar Wiimote berfungsi di osu!, Kamu mungkin harus memeriksa opsi yang ditunjukkan pada gambar di atas.
```
Comment (wowcake) [inline wiki/Gameplay/Input_device/Wiimote/id.md:15]:
```suggestion
Agar Wiimote dapat berfungsi di osu!, Kamu mungkin harus menyalakan opsi yang ditunjukkan pada gambar di atas.
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Input_device/Beatmania_controller/id.md (hunk 1 above)
Comment (mousesalt) [reply wiki/Gameplay/Input_device/Beatmania_controller/id.md:9]: the id page doesn't exist yet. keeping this as original now

File: wiki/Gameplay/Input_device/Dance_pad/id.md (hunk 5 above)
Comment (mousesalt) [reply wiki/Gameplay/Input_device/Dance_pad/id.md:9]: the ID page is outdated

Comment (wowcake) [reply wiki/Gameplay/Input_device/Dance_pad/id.md:9]: fair enough

**Review by wowcake — APPROVED** [review]
good to go

**Review by TicClick — APPROVED** [review]

**Pull request merged**